game.py

```diff
@@ -20,8 +20,29 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
 
+    print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        guessed_letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if guessed_letter in correct_letter_guess_statuses:
+            correct_letter_guess_statuses[guessed_letter] = True
+        else:
+            wrong_guesses_list.append(guessed_letter)
+            wrong_guesses_count += 1
+            print_snowman_graphic(wrong_guesses_count)
+        
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+            print("Congratulations! You win!")
+            return
```

Comment:
What's another way the logic in this loop could be written so that you can end the game without needing to use `return` to break the loop?